docs/pages/access-controls/guides/role-templates.mdx

```diff
@@ -281,6 +281,8 @@ spec:
 
     # Functions transform variables.
     database_users: ['{{email.local(external.email)}}']
+    database_labels:
+      'env': '{{regexp.replace(external.acces["env"], "^(staging)$", "$1")}}']
```

Comment:
Is there a dangling `]` here?

Reply:
Yeah my bad, a couple typos slipped through here actually, should be fixed in the latest commit